examples/projects/NUCLEO-F072RB/gate_serialcom/src/main.c

```diff
@@ -83,7 +88,19 @@ int main(void)
     SystemClock_Config();
 
     /* USER CODE BEGIN SysInit */
+#ifdef BOOTLOADER_MODE
+    /* Copy the vector table from the Flash (mapped at the base of the application
+        load address 0x0800C800) to the base address of the SRAM at 0x20000000. */
+    for (uint32_t i = 0; i < 48; i++)
+    {
+        VectorTable[i] = *(__IO uint32_t *)(0x0800C800 + (i << 2));
```

Comment:
put a define here for the adresse 0x800C8000

Reply:
OK